pydarn/plotting/superdarn_radars.py

```diff
@@ -1,101 +1,116 @@
-# Copyright (C) 2019 SuperDARN
-# Author: Marina Schmidt
+# Copyright (C) SuperDARN Canada, University of Saskatchewan
+# Authors: Marina Schmidt and Danno Peters
 
 """
 This module contains SuperDARN radar information
 """
 from typing import NamedTuple
 from enum import Enum
+from datetime import datetime
 
-
+"""
+_Hemispheres Class - Used to denote which hemisphere a radar is located in
+      _Hemisphere.North = 1
+      _Hemisphere.South = 0
+"""
 class _Hemisphere(Enum):
     North = 0
     South = 1
+    
 
+"""
+_Coord Class - Used to store Geo Coordinates (utilized for both GeoGraphic and GeoMagnetic)
+      Nested inside of _Radar Named Tuple, composed of three floats representing location and boresight direction
+
+      lat: Latitude in decimal format
+      lon: Longitude in decimal format
+      bore: Boresight in decimal format
+"""
+class _Coord(NamedTuple):
+  lat: float
+  lon: float
+  bore: float
 
+"""
+_Radar Class - Used to store relevent information about the SuperDARN Radars
+      Nested inside of _Radar Named Tuple, composed of three floats representing location and boresight direction
+      
+      name: full text of radar name
+      stid: station number
+      abbrev: three letter station abreviation
+      beams: number of possible beams
+      gates: number of range gates per beam
+      geographic: Named Tuple containing geographic latitude longitude and boresight
+      geomagnetic: Named Tuple containing geomagnetic latitude longitude and boresight
```

Comment:
What about altitude?

Reply:
Where do you get altitude values? 
@DannoPeters got this information from VT. 

Maybe we can add altitude in later?

Reply:
I believe we need more accurate location data (VT's resolution is 2 decimal or 1.1km) before we can get accurate elevation data.

I think the highest resolution data set which covers all the radar sites would be the NASA Shuttle LiDAR (30m resolution and 16m accuracy). Here is a repo for parsing the elevation data set in python: https://github.com/tkrajina/srtm.py

Reply:
The VT radar table has been known to be incorrect many times in the past due to its reliance on manual updates.  The definitive source of radar latitude, longitude, and altitudes are the hdw.dat files, which provide altitude to an accuracy of a tenth of a meter (although you may only want to trust it to the nearest meter or so depending on the radar).  Furthermore, the geomagnetic latitudes/longitudes/boresites in the VT radar table are most certainly incorrect as those were found using the 2005 epoch of AACGM coefficients, which have been shown to be inaccurate by [Shepherd [2014]](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2014JA020264).

Reply:
Created Issue #23 for this. Further discussion needs to be placed there.